integration_tests/androidx_test/build.gradle.kts

```diff
@@ -0,0 +1,77 @@
+plugins {
+  alias(libs.plugins.android.library)
+  alias(libs.plugins.robolectric.android.project)
+  alias(libs.plugins.robolectric.gradle.managed.device)
+}
+
+android {
+  compileSdk = 34
+  namespace = "org.robolectric.integration.axt"
+
+  defaultConfig {
+    testApplicationId = "org.robolectric.integrationtests.axt"
+    minSdk = 21
+    testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    testInstrumentationRunnerArguments["useTestStorageService"] = "true"
+  }
+
+  compileOptions {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+  }
+
+  testOptions {
+    targetSdk = 34
+    unitTests.isIncludeAndroidResources = true
+  }
+
+  sourceSets {
+    val sharedTestDir = "src/sharedTest/"
+    val sharedTestSourceDir = sharedTestDir + "java"
+    val sharedTestResourceDir = sharedTestDir + "resources"
+    val sharedAndroidManifest = sharedTestDir + "AndroidManifest.xml"
+
+    val test by getting
+    test.resources.srcDirs(sharedTestResourceDir)
+    test.java.srcDirs(sharedTestSourceDir)
+    test.manifest.srcFile(sharedAndroidManifest)
+
+    val androidTest by getting
```

Comment:
`by getting` will look for a source set named `androidTest`. If it does not find it, it will throw an `UnknownDomainObjectException` exception at compile time. I just renamed the variable to `androidTest2` to check the behavior.
I suppose that the `android-library` plugin adds this source set.